src/executors/public_1559_executor.rs

```diff
@@ -110,65 +91,79 @@ where
+                                        send_metric_with_order_hash!(&order_hash, metric_future);
+                                    }
+                                    Err(anyhow::anyhow!("Order Already Filled"))
+                                }
+                                ReactorErrorCode::InvalidDeadline => {
+                                    info!(
+                                        "{} - Order past deadline, skipping execution",
+                                        order_hash
+                                    );
+                                    let metric_future = build_metric_future(
+                                        self.cloudwatch_client.clone(),
+                                        DimensionValue::PriorityExecutor,
+                                        CwMetrics::ExecutionSkippedPastDeadline(chain_id_u64),
+                                        1.0,
+                                    );
+                                    if let Some(metric_future) = metric_future {
+                                        send_metric_with_order_hash!(&order_hash, metric_future);
+                                    }
+                                    Err(anyhow::anyhow!("Order Past Deadline"))
+                                }
+                                _ => Ok(1_000_000),
                             }
-                            Err(anyhow::anyhow!("Order Already Filled"))
+                        } else {
+                            warn!("{} - Unexpected error data: {:?}", order_hash, serde_value);
+                            Ok(1_000_000)
                         }
-                        ReactorErrorCode::InvalidDeadline => {
-                            info!("{} - Order past deadline, skipping execution", order_hash);
-                            let metric_future = build_metric_future(
-                                self.cloudwatch_client.clone(),
-                                DimensionValue::PriorityExecutor,
-                                CwMetrics::ExecutionSkippedPastDeadline(chain_id_u64),
-                                1.0,
-                            );
-                            if let Some(metric_future) = metric_future {
-                                send_metric_with_order_hash!(&order_hash, metric_future);
-                            }
-                            Err(anyhow::anyhow!("Order Past Deadline"))
-                        }
-                        _ => Ok(U256::from(2_000_000)),
+                    } else {
+                        warn!("{} - Failed to parse error data: {:?}", order_hash, err);
+                        Ok(1_000_000)
                     }
                 } else {
-                    warn!("Error estimating gas: {:?}", err);
-                    Ok(U256::from(2_000_000))
+                    warn!("{} - Error estimating gas: {:?}", order_hash, err);
+                    Ok(1_000_000)
```

Comment:
was decreasing this intentional?

Reply:
let's make 1_000_000 a constant so it's not copy-pasta'd everywhere

Reply:
the decrease is intentional -- actually gas usage is way lower